drivers/at30ts74/at30ts74.c

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (C) 2016 University of California, Berkeley
+ * Copyright (C) 2016 Michael Andersen <m.andersen@cs.berkeley.edu>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_at30ts74
+ * @{
+ *
+ * @file
+ * @brief       Driver for the AT30TS74 ±1°C Accurate, 12-Bit Digital I2C
+ *              Temperature Sensor from Atmel
+ *
+ * @author      Michael Andersen <m.andersen@cs.berkeley.edu>
+ *
+ * @}
+ */
+
+#include "at30ts74.h"
+#include "periph/i2c.h"
+#include <xtimer.h>
+
+#define ENABLE_DEBUG    (0)
+#include "debug.h"
+
+#define CFG_BITS_SHIFT 5
```

Comment:
Since you were using parens for other defines, I would vote for consistency. (Sorry for the nitpick.)